node/src/Commands.ts

```diff
@@ -3855,6 +3855,7 @@ export interface BaseScanOptions {
  */
 export interface ScanOptions extends BaseScanOptions {
     type?: ObjectType;
+    allowNonCoveredSlots?: boolean;
```

Comment:
Add docs to both fields

Reply:
type have doc here above. so just the last.

Reply:
But theres an important point here that you help me discover.
Thats not ScanOption filed, thats ClusterScanOption filed!